modules/Relation.tla

```diff
@@ -32,6 +32,11 @@ IsAsymmetric(R, S) == \A x,y \in S : ~(R[x,y] /\ R[y,x])
 (***************************************************************************)
 IsTransitive(R, S) == \A x,y,z \in S : R[x,y] /\ R[y,z] => R[x,z]
 
+(***************************************************************************)
+(* Is the relation R comparable over the set S?                            *)
```

Comment:
The wording of the comment sounds a little strange to me: what does it mean for a relation to be "comparable"? Perhaps "Does R relate all elements of set S" or something along these lines?